modules/core/arrow-data/src/main/kotlin/arrow/data/Store.kt

```diff
@@ -0,0 +1,16 @@
+package arrow.data
+
+import arrow.higherkind
+
+@higherkind
+data class Store<S, V>(val state: S, private val render: (S) -> V) : StoreOf<S, V> {
```

Comment:
Why make render private here?

Reply:
I made it private because I see no sense in allowing the user to use that function with any other state than the one of that Store and we have `extract` for that case.

Reply:
I don't think we're making properties in Kleisli or Reader/Writer/State, so it's okay to keep them public.